static/src/stylesheets/module/identity/_identity.scss

```diff
@@ -8,6 +8,41 @@
     }
 }
 
+.identity-wrapperForBg {
```

Comment:
Small request: by convention we're using `hyphen-delimited` naming for CSS classes rather than `camelCase`. I'm not sure why this isn't currently being linted against!

Reply:
@SiAdcock actually is there a readme.md detailing the conventions? because i mostly made up a lot of it from the surrounding code and I'll admit that's not the best idea

Reply:
Fair point. We have a very out of date [CSS styleguide](https://github.com/guardian/pasteup/blob/master/less/README.md). I see now the surrounding code doesn't follow the conventions, so apologies for that, also. It looks like the `identity` module is an outlier with its own special conventions ✨ 

In the interests of pragmatism, let's leave it for now. I'll make a note to tidy it up, which will likely end up at the very bottom of a priority list.

Reply:
hahaha that's fair. a lot of the new css on the pr is namespaced under .manage-account rather than .identity (whole thing is a mess really) so i'll try to make sure _that's_ up to standards before merging and hopefully make some sense of the whole thing in the future!

Reply:
That will be very helpful, thank you!